setup.py

```diff
@@ -194,6 +195,7 @@ def write_version(filename=os.path.join(*['airflow',
     'rednose',
     'paramiko',
     'pysftp',
+    'pywinrm',
```

Comment:
Hmn.. looks like the others are no in alphabetic ordering.. can you possibly fix these?

Reply:
fixed, commit coming